doc/manifest/schema/1.0.0/README.md

```diff
@@ -1,7 +1,7 @@
 [versionSchema]:        https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.version.1.0.0.json
 [defaultLocaleSchema]:  https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.defaultLocale.1.0.0.json
 [localeSchema]:         https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.locale.1.0.0.json
-[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.0.0.json
+[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json
```

Comment:
```suggestion
[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.installer.1.0.0.json
```

Reply:
Oops, I assumed the previous commitment to point it toward 1.1.0. Thanks for the sharp eyes, @aisgbnok

Reply:
Ty.